### PR TITLE
master

### PR DESCRIPTION
**Pull request by guijesuis**
my first PR to you

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
+    "android": "node node_modules/react-native/local-cli/cli.js run-android",
+    "http": "adb remote tcp:8081 tcp:8081 && adb remote tcp:8081 tcp:8081",
```
Comment (totorototo) [inline package.json]: adb remote tcp:8081 tcp:8081 x2?

File: package.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
+    "android": "node node_modules/react-native/local-cli/cli.js run-android",
```
Comment (totorototo) [inline package.json:7]: iOS?

**Review by totorototo — COMMENTED** [review]
comments / questions about added scripts

File: package.json (hunk 1 above)
Comment (guijesuis) [reply package.json]: good u saw the mistake. it was abviously meant to be adb reverse twice

File: package.json (hunk 2 above)
Comment (guijesuis) [reply package.json:7]: why not?

Comment (totorototo) [reply package.json:7]: waiting for you to add iOS script!

File: package.json (hunk 1 above)
Comment (totorototo) [reply package.json]: so fix it :p

**Review by totorototo — DISMISSED** [review]
this needs to be fixed before being merged.

**Comment by totorototo** [comment]
merge pr

**Pull request merged**